src/autorest-core/lib/configuration.ts

```diff
@@ -37,6 +37,8 @@ export interface AutoRestConfigurationImpl {
   "fluent"?: boolean;
   "azure-arm"?: boolean;
```

Comment:
@olydis why does AutoRest still need this flag? to determine whether to run the validator? If yes, I think it should be renamed to something more meaningful, given we are making these changes.